Makefile

```diff
@@ -14,24 +14,35 @@ build-js-min: ## Build /skycoin/skycoin.go. The result is minified and saved in
 	go build -o gopherjs-tool vendor/github.com/gopherjs/gopherjs/tool.go
 	GOOS=linux ./gopherjs-tool build skycoin/skycoin.go -m -o js/skycoin.js
 
+build-wasm: ## Build /wasm/skycoin.go. The result is saved in the repo root as skycoin-lite.wasm
+	GOOS=js GOARCH=wasm go build -o js/skycoin-lite.wasm ./wasm/skycoin.go
+
 test-js: ## Run the Go tests using JavaScript
 	go build -o gopherjs-tool vendor/github.com/gopherjs/gopherjs/tool.go
 	./gopherjs-tool test ./skycoin/ -v
 
-test-suite-ts: ## Run the ts version of the cipher test suite. Use a small number of test cases
+test-suite-ts: ## Run the ts version of the cipher test suite for GopherJS. Use a small number of test cases
 	cd js && npm run test
 
-test-suite-ts-extensive: ## Run the ts version of the cipher test suite. All the test cases
+test-suite-ts-extensive: ## Run the ts version of the cipher test suite for GopherJS. All the test cases
 	cd js && npm run test-extensive
 
+test-suite-ts-wasm: ## Run the ts version of the cipher test suite for wasm and additional tests
+	cd vendor/github.com/skycoin/skycoin/src/cipher/secp256k1-go && GOOS=js GOARCH=wasm go test -c -o test.wasm
```

Comment:
Can we run these tests for `cipher/` as well?

Reply:
Yes, no problem. Done in the lastest commit